pageComponents/NewPostPage/PlayerLengthSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Select, { Option } from "@@/components/Select";
+
+interface Props extends React.Attributes {
+  onChange?: (playerLength: number) => void;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PlayerLengthSelect({ onChange, ...props }: Props) {
+  return (
+    <Select
+      onChange={(_, value) => onChange && onChange(Number(value))}
```

Comment:
A lot of programmers use `&&` for this kind of usage, but it evaluates as bool value, which means it's semantically inappropriate. Please use `if` or `??`, or set default value.

```suggestion
export default function PlayerLengthSelect({ onChange = () => {}, ...props }: Props) {
  return (
    <Select
      onChange={(_, value) => onChange(Number(value))}
```

Reply:
why is it better?